docs/configure.rst

```diff
@@ -0,0 +1,131 @@
+.. highlight:: console
+
+
+.. _configure:
+
+=========
+Configure
+=========
+
+This document shows you how to install `Croud`_, the
+`command-line interface (CLI)`_ tool for CrateDB Cloud. Croud runs in your
+computer's terminal application. It is designed to manage CrateDB Cloud
+functions automated and text-based.
+
+You'll also learn how to log into CrateDB Cloud and access the `Cloud Console`_.
+The Cloud Console is a web administration tool with a graphical user interface
+that you can use along with the CLI to interact with CrateDB Cloud. The Console
+provides you with comprehensive information about the CrateDB clusters you have
+deployed.
+
+.. NOTE::
+
+  This section assumes that you have `created a CrateDB Cloud account`_.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Install Croud
+=============
+
+For the Croud CLI to run correctly, you need to install `Python 3.6 or higher`_
+on your system.
+
+You have Python installed.
+
+Next, install the Croud CLI:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Check if the installation was successful, like so:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+The command returns the current version number of your Croud CLI installation:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+.. NOTE::
+
+  Make sure to always `update Croud to the latest version`_ to ensure that all
+  feature developments are included.
+
+.. _sign-in:
+
+Log in
+======
+
+Logging in to CrateDB Cloud is a two-step process.
+
+First, run this command in Croud:
+
+.. code-block:: console
+
+    sh$ croud login
+
+If successful, you should get this message:
+
+`You have successfully logged into CrateDB Cloud!`
```

Comment:
it's displayed in the browser window. the screen only displays two sentences in the top-left corner and otherwise, the screen is empty so i haven't included a screenshot